homeassistant/components/litejet/light.py

```diff
@@ -46,17 +47,19 @@ class LiteJetLight(LightEntity):
     _attr_should_poll = False
     _attr_supported_color_modes = {ColorMode.BRIGHTNESS}
     _attr_supported_features = LightEntityFeature.TRANSITION
+    _attr_has_entity_name = True
+    _attr_name = None
```

Comment:
Typically, only a single entity (the device's "main entity") should set `_attr_name` to `None`.
Since light entities are created in a loop, does that hold true?

Reply:
Yup. As in the other comment think of LiteJet like a hub that is exposing multiple physical devices the user interacts with.